### PR TITLE
Enables login

### PR DESCRIPTION
**Pull request by marcelo140**
This PR adds the following changes:
- Adds login/logout button to the page
- Removes facebook authentication as requested
- Redirects users to the welcome page if they try to sign in while authenticated

**Question:** To remove the facebook authentication I guided myself by https://github.com/cesium/atomic/pull/52/ where there was a discussion about the `auth_email` in the user model. It was stated that the method was needed since the user could choose not to share his email when logging in with facebook. Is this method needed for github/email authentication which were added later? Can the users opt to not share the email with these two providers? 

**Note:** This PR should not be merged to master until we confirm that the activities are working as intended.

### CODE REVIEW

**Comment by martinhoaragao** [comment]
@joaofcosta could you take a look at the auth_email that he mentioned in the PR description? 
I think it can be removed, as I can only recall Facebook allowing one to not provide the email. Everything other than that seems rather fine.

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class SessionsController < ApplicationController
+  def new
+    redirect_to root_url unless session[:user_id].nil?
```
Comment (fribmendes) [inline app/controllers/sessions_controller.rb]: I'd rather have this done with `pandit` or `cancancan`, tbh

Comment (marcelo140) [reply app/controllers/sessions_controller.rb]: The project uses `cancancan` but wouldn't I need to have a `session` model to use it here?

Comment (fribmendes) [reply app/controllers/sessions_controller.rb]: Two options:

1. which I'm not really a fan of:

```ruby
before_filter -> { raise CanCan::AccessDenied if current_user }
```

2. What's described in here: https://github.com/ryanb/cancan/issues/22

**Comment by marcelo140** [comment]
Regarding to the `auth_email` method, I checked our current database and @9thScientist, who logged in with github, has an email provided by this method which is even weirder since he has a public email on github.

**Comment by fribmendes** [comment]
@marcelo140 not on par with what is happening but don't forget GitHub allows private emails as well.

**Pull request merged**